data-transfer-service/src/main/java/com/epam/pipeline/dts/common/service/CloudPipelineAPIClient.java

```diff
@@ -77,4 +81,17 @@ public static CloudPipelineAPIClient from(final PipelineCredentials credentials)
                 .map(data -> data.get(key))
                 .map(PipeConfValue::getValue);
     }
+
+    public Optional<DtsRegistry> tryLoadDtsRegistryByNameOrId(final String dtsId) {
```

Comment:
What about calling this method `findDtsRegistryByNameOrId`?

Reply:
Done, renamed